### DISCUSSION TITLE
Support Request: error from dotnet when when trying to run otd-gui

### DISCUSSION THREAD

**Discussion by taaigo** [OpenTabletDriver/OpenTabletDriver#3210] [General]
### What do you need help with?

I installed otd on my Gentoo system through the guru overlay.
The daemon works fine but running otd-gui gives this error message:
```
/usr/bin/otd-gui: line 2: cd: /usr/share/OpenTabletDriver: No such file or directory
Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET program, but dotnet-./OpenTabletDriver.UX.Gtk.dll does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
```

### What have you tried?

re-emerging the package. For the rest i don't know what else there is to do, it's pointing to files that should be there but aren't.
I couldn't find much about it anywhere else on the internet.

### Diagnostic Information

N/A

### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [x] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.
- [X] I am on the latest version of OpenTabletDriver.

**Comment by jamesbt365** [comment]
The `otd` and `otd-gui` scripts in the ebuild point at the wrong directory when trying to invoke the driver, you should report this to to the respective place so that the package maintainer (ethannij) can deal with it.

Looking further you should probably use the source package instead of the bin package, its better maintained.

**Comment by S41G0N** [comment]
I know it's not the best solution, but uninstalling OpenTabletDriver-bin and emerging OpenTabletDriver instead will work. It's the easiest workaround until the issue is fixed.
